cmd/admin-service-restart.go

```diff
@@ -144,22 +152,22 @@ func mainAdminServiceRestart(ctx *cli.Context) error {
 	t := time.Now()
 	for {
 		select {
-		case <-globalContext.Done():
-			return globalContext.Err()
+		case <-ctxt.Done():
+			return ctxt.Err()
 		case <-timer.C:
-			ctx, cancel := context.WithTimeout(globalContext, 3*time.Second)
-			// Fetch the service status of the specified MinIO server
-			info, e := client.ServerInfo(ctx)
-			cancel()
+			healthCtx, healthCancel := context.WithTimeout(ctxt, 3*time.Second)
+			// Fetch the health status of the specified MinIO server
+			healthResult, healthErr := anonClient.Healthy(healthCtx, madmin.HealthOpts{ClusterRead: true})
```

Comment:
We should only use Write since that is what everyone is interested

Reply:
> We should only use Write since that is what everyone is interested

Yes, we don't have an API for that yet. I can add '/minio/health/cluster/write' for that purpose, what do you think?

Reply:
If you don't have `ClusterRead` set then its Write @vadmeste `/minio/health/cluster` returns 200 OK if there is write quorum.

Reply:
> If you don't have ClusterRead set then its Write @vadmeste /minio/health/cluster returns 200 OK if there is write quorum.

Yes right, updated the PR